src/components/Input/Input.tsx

```diff
@@ -0,0 +1,39 @@
+import cn from "classnames";
+
+import "./input.scss";
+
+type Props = {
+  placeholder: string;
+  name: string;
+  value: any;
+  setInputState: any;
+  variant: string;
+};
+
+export default function Input({
+  placeholder,
+  name,
+  value,
+  setInputState,

```

Comment:
let's covert it to the name of the method/event, we don;t need to tell what it will do.

 ```suggestion
  onChange,
```